cookbooks/Gemfile

```diff
@@ -1,13 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'chef', '~> 11.18.12'
+gem 'chef'
+# Use solve 2.0 to remove gecode dependency (cf. berkshelf/berkshelf#1475)
+gem 'berkshelf', github: 'berkshelf/berkshelf', ref: 'a05e392'
 
 # Test-kitchen fork includes several helpful PRs not merged in the official repo:
-# 845 (ssh_tgz transport)
-# 825 (fix zlib warning)
 # 564 (multiple step converge)
-# 824 (fix chunked output)
 gem 'test-kitchen', github: 'wjordan/test-kitchen', branch: 'cdo'
 gem 'kitchen-docker'
-# Use with solve 2.0 to remove gecode dependency (cf. berkshelf/solve#46)
-gem 'berkshelf', github: 'wjordan/berkshelf'
+gem 'kitchen-ec2', github: 'wjordan/kitchen-ec2', branch: 'cdo'
```

Comment:
Could you add a comment explaining why we need a private fork of kitchen-ec2?   Would it be possible to submit back your changes so that we aren't running on a fork?